node_test.go

```diff
@@ -856,9 +778,7 @@ func TestAppendPagination(t *testing.T) {
 			for _, e := range m.Entries {
 				size += len(e.Data)
 			}
-			if size > maxSizePerMsg {
-				t.Errorf("sent MsgApp that is too large: %d bytes", size)
-			}
+			assert.LessOrEqual(t, size, maxSizePerMsg, "sent MsgApp that is too large: %d bytes", size)
```

Comment:
```suggestion
			assert.LessOrEqual(t, size, maxSizePerMsg, "sent MsgApp that is too large")
```

An error message will already print the `size` and `maxSizePerMsg`, so no need to add it second time.